### DISCUSSION TITLE
Procedure of training a HiFiGan Vocoder with custom data set

### DISCUSSION THREAD

**Discussion by michaellin99999** [coqui-ai/TTS#1034] [General Q&A]
Hi, I cant find any documentation on how to train a HiFiGan vocoder.
From the papers, is the input data for training spectrograms produced by TTS model? and evaluation is done against the ground true audio files of the input data? 

Is my reasoning correct where if I want to train a HiFiGan vocoder using LJSpeech dataset, I would basically need to produce spectrograms using a TTS model for the data in LJSpeech and then feed the sepctrograms alongside the original audio into the vocoder?

Thanks

**Comment by erogol** [comment]
https://tts.readthedocs.io/en/latest/tutorial_for_nervous_beginners.html#training-a-vocoder-model

**Comment by betimd** [comment]
I trained a hifigan vocoder but when I try to load checkpoints on inference I get this error:

RuntimeError: Given groups=1, weight of size [512, 80, 7], expected input[1, 201472, 11] to have 80 channels, but got 201472 channels instead

any ideas?

**Reply by zaydenyang** [reply]
Encountered the same problem
